MAUI/Polar-Charts/Axis/Ticklines.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: Axis Tick Line in .NET MAUI Chart control | Syncfusion
+description: Learn here all about the chart axis tick lines and their customization in the Syncfusion .NET MAUI Chart (SfPolarChart) control.
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Tick Lines in .NET MAUI Chart
+
+Tick lines are the small lines which is drawn on the axis line representing the axis labels. By default, tick lines are drawn outside of the axis.
+
+Minor tick lines can be added to the axis by defining the [MinorTicksPerInterval]() property. This property will add the minor tick lines to every interval based on value.
+
+N> For category axis, minor tick lines are not applicable as it is rendered based on index positions.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+. . .
+<chart:SfPolarChart.PrimaryAxis>
+    <chart:NumericalAxis />
+</chart:SfPolarChart.PrimaryAxis>
+
+<chart:SfPolarChart.SecondaryAxis>
+    <chart:NumericalAxis MinorTicksPerInterval="4"/>
+</chart:SfPolarChart.SecondaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+NumericalAxis primaryAxis = new NumericalAxis()
+chart.PrimaryAxis.Add(primaryAxis);
+
+NumericalAxis secondaryAxis = new NumericalAxis()
+{
+    MinorTicksPerInterval = 4 
+};
+chart.SecondaryAxis.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customization
+
+Both major and minor tick lines can be customized by using the [MajorTickStyle]() and [MinorTickStyle]() properties respectively. They provide options to change the [StrokeWidth](), [TickSize](), [Stroke]() of tick lines. By default minor tick lines will not be visible.
```

Comment:
Both major and minor tick lines can be customized using the [MajorTickStyle]() and [MinorTickStyle]() properties respectively. They provide options to change the [StrokeWidth](), [TickSize](), [Stroke]() of tick lines. By default, minor tick lines will not be visible.